styles.css

```diff
@@ -96,7 +96,8 @@ footer p {
 }
```

Comment:
Please add a link to the corresponding line of code in the Pull Request If any issues are found and changes are made.

Reply:
Hello @SajjadAhmad14 ,

Thank you for the insightful review 🤓. We have added the links to the corresponding lines as per guide, in our pull request description.

> HappyCoding